### PR TITLE
pytorch: 0.1.12 -> 0.2.0

### PR DESCRIPTION
**Pull request by ericsagnes**
###### Motivation for this change

Update to latest version.

cc maintainer @teh 

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ericsagnes** [comment]
Note: Added torchvision

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27064,6 +27064,28 @@ EOF
     };
   };
 
+  torchvision = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: can you move this to a separate file as describes in the header of this file

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -27064,6 +27064,28 @@ EOF
...
+    version = "0.1.9";
+    pname   = "torchvision";
+    name    = "${pname}-${version}";
+
+    format = "wheel";
+
+    src = pkgs.fetchurl {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: fetchPypi

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (ericsagnes) [inline pkgs/top-level/python-packages.nix]: Thanks for the review! Pushed fixes.

**Pull request merged**